openlibrary/macros/EditButtonsMacros.html

```diff
@@ -13,7 +13,7 @@
 </div>
 
 <div class="cclicense">
-    $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="$_('This link to Creative Commons will open in a new window')">$_("CC0")</a>. Yippee!')
+    $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="$_('This link to Creative Commons will open in a new window')">CC0</a>. Yippee!')
```

Comment:
@tabshaikh this still has nested `$_`s   ` title="$_('This link to Creative Commons will open in a new window')` is inside `$:_('By saving ... Yippee!')`

We need to form some guidelines about including markup in i18n-able strings. I thought it was bad practice to include page markup in the strings -- although there may be a way to provide context to help with translation.

```
$:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under) <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="$_('This link to Creative Commons will open in a new window')">CC0</a>. 
```
Would be my suggestion, but I'm not sure how to indicate the end of that sentence to a translator. It's further complicated by the fact that are different links to CC licences in other languages. I tested and the CC0 page does not automatically select the language based on browser settings, like OL is supposed to do.

@tfmorris, what are your thoughts here?

Reply:
@hornc I have updated the pr we could discuss in the slack channel about guidelines to include markup of i18n-able string and waiting for the feedback of @tfmorris